great_expectations/data_context/store/namespaced_read_write_store.py

```diff
@@ -180,6 +180,7 @@ def __init__(self,
                  serialization_type=None,
                  store_backend=None
                  ):
+        self.key_class = SiteSectionIdentifier
```

Comment:
This should be a class attribute instead of an instance attribute (just thinking about generally making our classes easier to subclass)